src/store/lineup/sagas.js

```diff
@@ -1,4 +1,4 @@
-import { delay } from 'redux-saga'
+import delay from '@redux-saga/delay-p'
```

Comment:
i think this should be now from @redux-saga/effects

https://redux-saga.js.org/docs/recipes/#debouncing

Reply:
nice catch, this was based off this change: delay became an effect, old delay function (not effect!) can be imported from @redux-saga/delay-p
so wasnt sure if switching from the function version to the effect version would break things?

Reply:
i think the interface is the same so probably alright?